shared/src/commonTest/kotlin/org/hyperskill/study_plan/screen/StudyPlanScreenTest.kt

```diff
@@ -46,9 +51,26 @@ class StudyPlanScreenTest {
         }
     }
 
+    @Test
+    fun `Retry content loading message should trigger logging analytic event`() {
+        val (_, actions) = reducer.reduce(
+            stubState(),
+            StudyPlanScreenFeature.Message.RetryContentLoading
+        )
+
+        assertEquals(actions.size, 4)
```

Comment:
I think we don't have a test that covers this - it would be nice to check if `toolbarState`, `problemsLimitState`, and `studyPlanWidgetState` has changed.